src/com/TTT/Main.java

```diff
@@ -12,14 +10,14 @@ private static void run() {
         Board board = new Board();
         BoardPrinter printer = new BoardPrinter(System.out);
         Game game = new Game(board);
-        TurnUI turns = new TurnUI();
+        TurnUI turns = new TurnUI(printer, System.out, System.in);
 
-        takeTurns(board, printer, game, System.in, turns);
+        takeTurns(board, game, turns);
     }
 
-    public static void takeTurns(Board board, BoardPrinter printer, Game game, InputStream in, TurnUI turns) {
+    public static void takeTurns(Board board, Game game, TurnUI turns) {
```

Comment:
I guess this is still not under test? Might be worth thinking about where the game loop should live.